src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -128,47 +133,43 @@
 
 ;; Filter history
 
-(defn- item-tokens [{:keys [symbol label checked?]}]
+(defn- item-filter [{:keys [icon checked? path]} content]
   [list/item
-   [list/item-icon (transaction-type->icon :pending)] ;; TODO(jeluard) add proper token data
+   [list/item-icon icon]
+   content
+   [list/item-checkbox {:checked? checked? :on-value-change #(re-frame/dispatch [:wallet.transactions/filter path %])}]])
+
+#_
```

Comment:
?

Reply:
Will be used when ERC20 support is added so I kept it.

Reply:
could you add a comment in the code?

Reply:
ok